### DISCUSSION TITLE
Dispatch App

### DISCUSSION THREAD

**Discussion by CptNoHand** [project-error/npwd#886] [Ideas]
Basically an App where people can call a number, like 911 and all police officers could answer the call and people can send a dispatch with the click of an button.
Yellow pages go unseen after restart..kinda meh.

**Comment by itschip** [comment]
I think Mojito has made this @CptNoHand

**Reply by CptNoHand** [reply]
Do you mean npwd-services?

If yes, the numbers are taken from each officer with name and his personal number. I don't like that when it comes to officers. Or is there something i've missed?
